dev/com.ibm.ws.jsf.2.3_fat/fat/src/com/ibm/ws/jsf23/fat/tests/JSF23CDIGeneralTests.java

```diff
@@ -477,39 +478,27 @@ public void testELImplicitObjectsInjectionWithNoFacesConfigAnnotation() throws E
      * @throws Exception
      */
     @Test
-    @SkipForRepeat(EE10_FEATURES) // Skipped due to HTMLUnit / JavaScript Incompatabilty (New JS in RC5)
     public void testFacesConverterBeanInjection() throws Exception {
-        try (WebClient webClient = new WebClient()) {
-
-            // Construct the URL for the test
-            String contextRoot = "ConverterValidatorBehaviorInjectionTarget";
-            URL url = JSFUtils.createHttpUrl(server, contextRoot, "index.xhtml");
-
-            HtmlPage page = (HtmlPage) webClient.getPage(url);
+        ExtendedWebDriver driver = new CustomDriver(new RemoteWebDriver(chrome.getSeleniumAddress(), new ChromeOptions().setAcceptInsecureCerts(true)));
+                String contextRoot = "ConverterValidatorBehaviorInjectionTarget";
+        String url = JSFUtils.createSeleniumURLString(server, contextRoot, "index.xhtml");
+        WebPage page = new WebPage(driver);
+        page.get(url);
+        page.waitForPageToLoad();
 
             // Verify that the page contains the expected messages.
-            assertTrue(page.asText().contains("JSF 2.3 support for injection into JSF Managed Objects"));
+            assertTrue(page.isInPage("JSF 2.3 support for injection into JSF Managed Objects"));
 
-            // Get the form that we are dealing with
-            HtmlForm form = page.getFormByName("form1");
+                WebElement input = page.findElement(By.id("form1:textId"));
+                input.sendKeys("Hello World");
 
-            // Get the input text and submit button
-            HtmlTextInput inputText = (HtmlTextInput) form.getInputByName("form1:textId");
-            HtmlSubmitInput submitButton = form.getInputByName("form1:submitButton");
+                page.findElement(By.id("form1:submitButton")).click();
 
-            // Fill the input text
-            inputText.setValueAttribute("Hello World");
+                driver.switchTo().alert().dismiss();
```

Comment:
Alert shows up on this page, so we need to dismiss it. Otherwise, selenium will complain and the test fails.